ACL/AclXmlFileloader.php

```diff
@@ -69,7 +69,8 @@ public function load(Module $module)
     protected function parseAcls(SimpleXMLElement $xml, Module $module)
     {
         //If there are no acl node continue to parse the xml
-        if (false === $acls = $xml->xpath('//config:acls/config:acl')) {
+        $acls = $xml->xpath('//config:acls/config:acl');
```

Comment:
You are right. It was more some kind of standardisation - feel free to dismiss it